File: android-exercises/10 - options_menu/OptionsMenu_java/app/src/main/java/com/bugrahankaramollaoglu/optionsmenu_java/MainActivity.java
package com.bugrahankaramollaoglu.optionsmenu_java;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.bugrahankaramollaoglu.optionsmenu_java.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {

        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);

    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater menuInflater = getMenuInflater();

        menuInflater.inflate(R.menu.options_menu, menu);

        return super.onCreateOptionsMenu(menu);
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {

        if (item.getItemId() == R.id.options_item2) {
            Intent intent = new Intent(this, MainActivity2.class);
            startActivity(intent);
        }

        if (item.getItemId() == R.id.options_item3) {
            Intent intent = new Intent(this, MainActivity3.class);
            startActivity(intent);
        }

        return super.onOptionsItemSelected(item);
    }
}

File: android-exercises/10 - options_menu/OptionsMenu_kotlin/app/src/main/java/com/bugrahankaramollaoglu/optionsmenu_kotlin/MainActivity.kt
package com.bugrahankaramollaoglu.optionsmenu_kotlin

import android.content.Intent
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


    }

    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
        val menuInflater = menuInflater

        menuInflater.inflate(R.menu.options_menu, menu)

        return super.onCreateOptionsMenu(menu)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {

        if (item.itemId == R.id.options_item2) {
            val intent: Intent = Intent(this@MainActivity, MainActivity2::class.java)
            startActivity(intent)
        } else if (item.itemId == R.id.options_item3) {
            val intent: Intent = Intent(this@MainActivity, MainActivity3::class.java)
            startActivity(intent)
        } else if (item.itemId == R.id.other1) {

        } else if (item.itemId == R.id.other2) {
            Toast.makeText(this@MainActivity, "aaa", Toast.LENGTH_SHORT).show()
        }

        return super.onOptionsItemSelected(item)
    }
}